src/python/devcontainer-feature.json

```diff
@@ -63,13 +63,12 @@
         "python.defaultInterpreterPath": "/usr/local/python/current/bin/python",
         "python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
         "python.formatting.blackPath": "/usr/local/py-utils/bin/black",
-        "python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
-        "python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
         "python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
+        "python.linting.flake8Enabled": false,
```

Comment:
Can we bump the patch version of this Feature? Thanks
https://github.com/devcontainers/features/blob/ad0b0cf5e9344131a0d72b614d138335f4bbd5b5/src/python/devcontainer-feature.json#L3

Reply:
How would you like the version number bumped: major, minor, or micro?

Reply:
We can bump as patch (ie micro)